.re-natal

```diff
@@ -1,7 +1,7 @@
 {
   "name": "StatusIm",
   "interface": "reagent",
-  "androidHost": "localhost",
+  "androidHost": "10.0.3.2",
```

Comment:
don't include this files

Reply:
it's still in progress, it doesn't suppose to be merged right now

Reply:
my bad, makes me realize the status we usually put in the PR description (ready/wip) is redundant with the column in zenhub